integration_test/EmitVerilog/lint.mlir

```diff
@@ -82,6 +82,7 @@ hw.module @TESTSIMPLE(%a: i4, %b: i4, %cond: i1, %array: !hw.array<10xi4>,
 
   %one = hw.constant 1 : i4
   %aPlusOne = comb.add %a, %one : i4
+  sv.verbatim "/* verilator lint_off WIDTH */"
   %29 = hw.array_slice %array at %aPlusOne: (!hw.array<10xi4>) -> !hw.array<3xi4>
```

Comment:
Can you also add an `sv.verbatim "/* verilator lint_on WIDTH */"` after this to turn it back on?

Reply:
IIRC, ops can be re-ordered around verbatim ops so this isn't guaranteed to work

Reply:
I'd be in favor of moving the test into its own file with a `RUN` command that is explicitly doing `-Wno-WIDTH` so that this is properly isolated.

Reply:
I agree with @teqdruid that this isn't guaranteed to work, but given the oddness of this case (it is clearly a verilator bug) and that we don't have a proper "attach an annotation comment to an expression design" yet, I think this is a practical step to unbreak the bots.

Reply:
Another option to consider is a style option to just not emit these expressions inline.

Reply:
The verbatim op actually gets reordered. I also added the `-Wno-WIDTH` for this module only.
As a followup, we might need to lower all packed arrays globally, if `disallowPackedArrays` is specified.